views/home.hbs

```diff
@@ -0,0 +1,23 @@
+<div id="addon">
+    <img class="logo" src="{{manifest.logo}}">
+    <h3 class="version"><i>{{manifest.version}}</i></h3>
+    <h3 class="description">{{manifest.description}}</h3>
+
+    <h2 class="gives">This add-on has:</h2>
+    <ul>
+        <li>
+            More
+            {{#each manifest.types}}
+                {{stylize this}}
+            {{/each}}
+            in Discover
+        </li>
+        <li>Watch for FREE</li>
```

Comment:
this is not true for all add-ons, and there is no way of saying that the streams are free or not in the manifest, i believe this line should be completely removed